external-crates/move/crates/move-analyzer/src/analysis/parsing_analysis.rs

```diff
@@ -209,13 +209,20 @@ impl<'a> ParsingAnalysisContext<'a> {
         let mut latest_use_loc = Loc::new(mod_def.loc.file_hash(), 0, 0);
         // location of the earliest member (if any)
         let mut earliest_member_loc = Loc::new(mod_def.loc.file_hash(), u32::MAX, u32::MAX);
+        // we need this to avoid adding auto-imports for `use` declarations
+        // in "use blocks" that follow the follow the initial one (and come
+        // after other module members)
```

Comment:
Is there a way to distinguish between `use fun` and `use member`? I believe it would be a cleaner solution.

Reply:
How would it work if I have a `use member` inside a function body?

Reply:
It only considers locations of `use` statements at the top (module) level as the auto-import location information is computed when traversing module members